verifiers/utils/dsse.go

```diff
@@ -19,6 +42,30 @@ func PayloadFromEnvelope(env *dsselib.Envelope) ([]byte, error) {
 	return payload, nil
 }
 
+func StatementFromPayload(payload []byte) (*intoto.Statement, error) {
```

Comment:
nit: be consistent with `EnvelopeFromBytes`
```suggestion
func StatementFromBytes(payload []byte) (*intoto.Statement, error) {
```